controller/tsconfig.json

```diff
@@ -31,6 +31,5 @@
   "include": [
     "next-env.d.ts",
     "**/*.ts",
-    "**/*.tsx"
-  ]
+    "**/*.tsx"]
```

Comment:
Looks like you removed a blank line that makes this array more readable on one of your edits.